WorkflowDemos/WorkflowDemosTest/src_test/test/bpm/TestProcurementRequest.java

```diff
@@ -0,0 +1,225 @@
+package test.bpm;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import ch.ivyteam.ivy.bpm.engine.client.BpmClient;
+import ch.ivyteam.ivy.bpm.engine.client.ExecutionResult;
+import ch.ivyteam.ivy.bpm.engine.client.UiMocker;
+import ch.ivyteam.ivy.bpm.exec.client.IvyProcessTest;
+import ch.ivyteam.ivy.process.model.value.PID;
+import ch.ivyteam.ivy.security.exec.Sudo;
+import ch.ivyteam.ivy.workflow.CaseState;
+import ch.ivyteam.ivy.workflow.ICase;
+import ch.ivyteam.ivy.workflow.ITask;
+import ch.ivyteam.ivy.workflow.TaskState;
+import workflow.humantask.ProcurementRequest;
+
+/**
+ * Early preview of {@link BpmClient} testing API usage. 
+ * 
+ * Here to expose difficulties when working with workflow centered processes.
+ * In order to improve until the release 9.1
+ *
+ * Tests the ProcurementRequest:
+ * <code>WorkflowDemos/processes/Humantask/ProcurementRequestParallel.mod</code>
+ * 
+ * @author rew
+ * @since Mar 3, 2020
+ */
+@SuppressWarnings("all")
```

Comment:
I guess this is a left-over ... or where are we doing something restricted in here?